src/lib/numericalInegrator/ERK.h

```diff
@@ -0,0 +1,98 @@
+#ifndef _ERK_H_
+#define _ERK_H_
+#include "Ode.h"
+
+class RK4 {
+	 public:
+    RK4 (
+	Derivs   f_,        // Differential equation
+	int      n_eqn_,    // Dimension
+	double   tInitial_,
+	double   tFinal_,
+	Mat      y0_,
+	double   h_,
+	void*    pAux_      // Pointer to auxiliary data	  		
+      ) 
+    : f_(f_), n_eqn_(n_eqn_),t_initial_(tInitial_),t_final_(tFinal_),y_0_(y_0_),h_(h_), pAux_(pAux_)
+    {
+		clock_t t1_ = clock();
+		k_1_=Mat(n_eqn_) , k_2_=k_1_ , k_3_=k_2_ , k_4_=k_3_;	

```

Comment:
Please have every statement in a separate line, and add spaces around the equal sign. Same for RK8.